src/probnum/diffeq/odefiltsmooth/kalman_odesolution.py

```diff
@@ -78,11 +78,15 @@ def __init__(self, kalman_posterior: pnfs.KalmanPosterior):
 
         # Pre-compute projection matrices.
         # The prior must be an integrator, if not, an error is thrown in 'GaussianIVPFilter'.
-        self.proj_to_y = kalman_posterior.gauss_filter.dynamics_model.proj2coord(
-            coord=0
+        self.proj_to_y = (
+            kalman_posterior.filter_posterior.gauss_filter.dynamics_model.proj2coord(
```

Comment:
the "visible" posterior (to the user) is always the smoothing posterior, and if nothing has been smoothed yet, the filterposterior can be accessed through `kalman_posterior.filter_posterior`.